tests/matchers.py

```diff
@@ -0,0 +1,59 @@
+"""
```

Comment:
Moved the file so you can just do `from tests.matchers import SomeMatcher` instead of `from tests.common.matchers`. The `tests/common/` subdir is just unnecessary and we should remove the remaining stuff (test factories) from it one day as well.